tools/cellxgene_census_builder/src/cellxgene_census_builder/build_state.py

```diff
@@ -48,6 +48,9 @@
     # Release clean up
     "release_cleanup_days": 32,  # Census builds older than this are deleted
     #
+    # Block list of dataset IDs
+    "dataset_id_blocklist_uri": "https://raw.githubusercontent.com/chanzuckerberg/cellxgene-census/main/tools/cellxgene_census_builder/dataset_blocklist.txt",
```

Comment:
Small concern: if the GitHub repo structure changes, this will likely break and might go unnoticed. Ideas:
1. Move to S3
2. Add it to the Docker image and read it from a file
3. Add a test that reads from this file specifically, so that in case it gets moved the tests will fail (this has to be done in a subsequent PR since the file isn't in main yet)

Reply:
Docker image solution could work. I know Bruce wanted to decouple builder releases from this data-only config: "the lifecycle of the block list is separated from the software release cycle and/or builder docker image build." But updating the blocklist file is going to create a new builder image anyway, right?

I considered the S3 solution as well during earlier Slack discussion, but felt we didn't have a great spot for it. I don't think we want it the Census bucket(s), if it ends up being visible to users, right?

Adding a test will only catch the file error at an arbitrary time, based upon development activity, which may not be in time for the next builder run. So probably not worth doing.

I'm also a tad concerned about Github file read design, but only because it makes our production system (builder) dependent on Github availability. It's not a great risk, but I've seen prod systems go down due to this. But since it's a batch process, I'm not as worried.

Overall, I'm okay with sticking with the design as is, but embedding in the Docker image seems fine to me as well and removes the runtime Github dependency.

One advantage of GitHub is the versioning of the file. S3 would provide versioning too, but not with changelogs and  authorship attribution.